### PR TITLE
Added free courses en for Neo4j Graph Database

### PR DESCRIPTION
**Pull request by rinaldipratama**
## What does this PR do?
Add resource(s)

### Description
Added free courses in the English language for Neo4j Graph Database.

### Why is this valuable (or not)?
It's really helpful for everyone who wants to learn Graph Database with Neo4j.

### How do we know it's really free?
Free on YouTube.

### For book lists, is it a book? For course lists, is it a course? etc.
It's a course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -240,6 +241,10 @@
 * [Introduction to Databases](https://lagunita.stanford.edu/courses/Engineering/db/2014_1/about) (Stanford University)
 * [Learn SQL Basic for Data Science Specialisation](https://www.coursera.org/specializations/learn-sql-basics-data-science#about) (Coursera)
 
+#### Neo4j
```
Comment (davorpa) [inline courses/free-courses-en.md:244]: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

Maybe in next... `No-SQL`?

**Comment by rinaldipratama** [comment]
wait, I will fix it

**Review by davorpa — CHANGES_REQUESTED** [review]
:information_source: Linter report:
```logs
Run fpb-lint ./courses/
/courses/free-courses-en.md
  239:1-242:141  warning  Incorrect number of blank lines between last section and next heading  blank-lines-1-0-2  remark-lint
   246:1-247:21  warning  Incorrect number of blank lines between last section and next heading  blank-lines-1-0-2  remark-lint

⚠ 2 warnings
Error: Process completed with exit code 1.
```

File: courses/free-courses-en.md (hunk 1 above)
Comment (rinaldipratama) [reply courses/free-courses-en.md:244]: oh, okay. I understand

**Comment by davorpa** [comment]
Work continues on #5832

**Pull request closed without merging**